addons/hot/functions/fnc_onFired.sqf

```diff
@@ -20,8 +20,8 @@ _stateParams params ["", "_seekerStateParams", "_attackProfileStateParams"];
 _seekerParams params ["", "", "_seekerMaxRange"];
 
 private _config = ([_projectile] call CBA_fnc_getObjectConfig) >> "ace_missileguidance";
-private _maxCorrectableDistance = if (isNumber(_config >> "correctionDistance")) then { getNumber(_config >> "correctionDistance") } else { DEFAULT_CORRECTION_DISTANCE };
-private _crosshairOffset = if (isArray(_config >> "offsetFromCrosshair")) then { getArray(_config >> "offsetFromCrosshair") } else { [0, 0, 0] };
+private _maxCorrectableDistance = [_config >> "correctionDistance", "number", DEFAULT_CORRECTION_DISTANCE] call CBA_fnc_getConfigEntry;
+private _crosshairOffset = [_config >> "offsetFromCrosshair", "array", [0, 0, 0]] call CBA_fnc_getConfigEntry;
```

Comment:
`CBA_fnc_getConfigEntry` for NUMBER entries does not support SQF expressions. The previous code didn't either. It should be written as:
```sqf
private _maxCorrectableDistance = DEFAULT_CORRECTION_DISTANCE;

if (isNumber (_config >> "correctionDistance") || {isText (_config >> "correctionDistance")}) then {
    _maxCorrectableDistance = getNumber (_config >> "correctionDistance");
};
```

Reply:
then maybe `CBA_fnc_getConfigEntry` should be fixed?

Reply:
@TheCandianVendingMachine could you please restore `CBA_fnc_getConfigEntry` since commy2 fixed that thing in CBA?

Reply:
Its not merged into CBA yet, so this code would be broken if I were to do that. Not to mention that if this gets merged and released before the next CBA update it could potentially cause issues for the end user.

Reply:
I don't think anybody would rush to add `NUMBER` config values to their missile guidance especially as SQF expressions. They are quite rare in configs and very rare in third-party addons. Also `correctionDistance` is not that value to be expression.
Anyway it's your decision.